### DISCUSSION TITLE
Tailwind generates inappropriate class names for custom colors

### DISCUSSION THREAD

**Discussion by yorgetsuten** [tailwindlabs/tailwindcss#13547] [Help]
I'm trying to add css vars aliases to the config like that:

`const config: Config = {
  theme: {
    colors: {
      '-bg-primary': 'var(--bg-primary)'
    }
  }
}`

Idea is to use them as bg--bg-primary, similar to css vars syntax. But tailwind generates a `-bg-bg-primary` class which contains `background-color: var(--bg-primary);`, instead of `bg--bg-primary` which does not seems like an expected behavior.

**Comment by wongjn** [comment]
I don't think you've missed something. There is precedent in Tailwind for having a leading `-` for negative values, like `-m-5`. So it could be that the key, `-bg-primary` is interpreted as a negative value, causing the first `-` to be moved to the start of the class name.

**Reply by yorgetsuten** [reply]
I guess it is a bug then
